adventureGame/AdventureGame/obj/DungeonRoom.py

```diff
@@ -52,9 +48,16 @@ def setHasEnemies(self, _hasEnemies):
     
     def setEnemyList(self, _enemyList):
         if type(_enemyList) is list:
-            for enemyListContent in _enemyList:
-                if isinstance(enemyListContent, EnemyClass.Enemy):
-                    self.enemyList = _enemyList
+            if len(_enemyList) == 0:
+                if debug == True:
+                    print("WWARNING: instance of dungeonRoom has an empty enemyList")
```

Comment:
Change print statement to:
print("WARNING: instance of dungeonRoom has an empty enemyList")

Reply:
Fixed.